sklearn/tests/test_discriminant_analysis.py

```diff
@@ -329,6 +360,60 @@ def test_lda_store_covariance():
     )
 
 
+def test_lda_shrinkage():
+    """
+    Test that shrunk covariance estimator and
+    shrinkage parameter behave the same
+    """
+    X = np.random.rand(100, 10)
+    Y = np.random.randint(3, size=(100))
+    c1 = LinearDiscriminantAnalysis(store_covariance=True, shrinkage=0.5,
+                                    solver="lsqr")
+    c2 = LinearDiscriminantAnalysis(
+            store_covariance=True,
+            covariance_estimator=ShrunkCovariance(shrinkage=0.5),
+            solver="lsqr")
+    c1.fit(X, Y)
+    c2.fit(X, Y)
+    for i in range(2):
+        np.testing.assert_allclose(c1.means_[i], c2.means_[i])
```

Comment:
I think that simply `np.testing.assert_allclose(c1.means_, c2.means_)` would work since it can compare array-likes?